src/screens/HomeScreen.tsx

```
@@ -55,17 +57,27 @@ const HomeScreen: React.FC = () => {
+                      </span>
+                    ))}
+                  </div>
                 </div>
                 <div className="column has-text-right">
                   <button
                     className="button is-danger is-light"
                     onClick={() => handleRemoveTrip(trip.id)}
-                  >Remove
+                  >
+                    <span className="icon">
+                      <i className="fas fa-trash"></i>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)